src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -57,102 +180,160 @@ const CreateBenchmarkPage = () => {
+                    }
+                    onChange={(e: ChangeEvent) =>
+                      setBenchmarkFormState({
+                        ...benchmarkFormState,
+                        ["number_of_trials"]: Number(e.target.value),
+                      })
+                    }
                   />
-                  <Typography ml={1}>%</Typography>
                 </Stack>
-                <TextField label="Metric to track" name="metric_name" />
-              </Stack>
-              <Stack direction="row" alignItems="center" spacing={7}>
-                <TextField
-                  label="Attempts Per Trial"
-                  type="number"
-                  name="attempts_per_trial"
-                />
-                <TextField
-                  label="Number of Trials"
-                  type="number"
-                  name="number_of_trials"
-                />
               </Stack>
             </Stack>
-          </Stack>
+          )}
+        </Stack>
 
-          <Stack spacing={4} p={3} pl={0} flexGrow={1}>
-            <Typography variant="h4">Details</Typography>
-            <TextField
-              multiline
-              label="Materials"
-              minRows={3}
-              required
-              name="materials"
-            />
-            <TextField
-              multiline
-              label="Instructions"
-              minRows={3}
-              required
-              name="instructions"
-            />
+        {viewState === VIEW_STATES.BENCHMARK_PG_1 && (
+          <Stack direction="row" spacing={2} p={4} pt={0} justifyContent="end">
+            <button
+              type="reset"
+              onClick={router.back}
+              className={$button.secondary}
+            >
+              Cancel
+            </button>
+            <button
+              onClick={() => setViewState(VIEW_STATES.BENCHMARK_PG_2)}
+              className={$button.default}
+            >
+              Next
+            </button>
           </Stack>
-        </Stack>
+        )}
 
-        <Divider />
-
-        <Stack direction="row" spacing={2} p={4} justifyContent="space-between">
-          <button
-            type="reset"
-            onClick={router.back}
-            className={$button.secondary}
-          >
-            Cancel
-          </button>
-          <button type="submit" className={$button.default}>
-            Create
-          </button>
-        </Stack>
+        {/* TO DO: Implement logic to check and see if required fields are filled out before allowing user to proceed to the next page */}
+        {viewState === VIEW_STATES.BENCHMARK_PG_2 && (
+          <Stack direction="row" spacing={2} p={4} justifyContent="end">
+            <button
+              onClick={() => setViewState(VIEW_STATES.BENCHMARK_PG_1)}
+              className={$button.secondary}
+            >
+              Back
+            </button>
+            <button type="submit" className={$button.default}>
```

Comment:
Suggestion here to go with line 59:

`<button type="submit" className={$button.default} disabled={!formComplete}>`